### PR TITLE
(feat): markdown linter

### PR DESCRIPTION
**Pull request by SimonvanWijhe**

### CODE REVIEW

File: .github/workflows/markdown-linting.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  pull_request:
+    paths:
+        - '**.md'
+
+jobs:
+  linting:
+    name: Markdown Linting
+    runs-on: ubuntu-latest
+    steps:
+      - uses: articulate/actions-markdownlint@v1
```
Comment (szepeviktor) [inline .github/workflows/markdown-linting.yml:12]: Try
```suggestion
      - uses: DavidAnson/markdownlint-cli2-action@v16.0.0
        with:
          globs: '**/*.md'
```

**Comment by szepeviktor** [comment]
All of my workflows: https://github.com/szepeviktor/byte-level-care/tree/master/.github/workflows

**Comment by SimonvanWijhe** [comment]
> All of my workflows: https://github.com/szepeviktor/byte-level-care/tree/master/.github/workflows

I’ll take a look at these. Thanks for sharing!

**Comment by szepeviktor** [comment]
I recommend you to leave the licence file alone: LICENSE.md -> LICENSE

**Comment by mvdhoek1** [comment]
> I recommend you to leave the licence file alone: LICENSE.md -> LICENSE

Hi @szepeviktor, can you explain your recommendation?

**Comment by szepeviktor** [comment]
1. the Markdown linter will not check it
2. everyone names that file without an extension
3. GitHub create `LICENSE` with "Propose" button https://github.com/yardinternet/skeleton-package/community

**Comment by mvdhoek1** [comment]
> 1. the Markdown linter will not check it
> 2. everyone names that file without an extension: [5.3M](https://github.com/search?q=path%3ALICENSE&type=Code) vs. [225k](https://github.com/search?q=path%3ALICENSE.md&type=code)
> 3. GitHub create `LICENSE` with "Propose" button https://github.com/yardinternet/skeleton-package/community

**Pull request merged**